app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
Since we are adding null constraints in the database, it's a good practice to also add them in the model so that the whole application is consistent.
An example of this could be, adding the following line:
```
validates :gender, :author, presence: true
```